src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -181,4 +179,34 @@ void CASEServer::OnSessionEstablished(const SessionHandle & session)
                     ChipLogValueScopedNodeId(session->GetPeer()));
     PrepareForSessionEstablishment(session->GetPeer());
 }
+
+void CASEServer::SendBusyStatusReport(Messaging::ExchangeContext * ec)
+{
+    using namespace Protocols::SecureChannel;
```

Comment:
I suggest a ChipLogError or ChipLogProgress here indicating that we are in the middle of another CASE handshake, hence responding with BUSY.

Reply:
Moved the below log here at the start